README.md

```diff
@@ -43,6 +43,7 @@ class Demo extends Component {
     super();
     this.state = {
       messageList: []
+      typing: true
```

Comment:
The line above this needs to end with a comma to be valid JS :)

Reply:
fixed ;)